content/en/blog/2021-03-08-flux-is-a-cncf-incubation-project.md

```diff
@@ -0,0 +1,108 @@
+---
+author: dholbach
+date: 2021-03-10 06:30:00+00:00
+title: Flux is a CNCF Incubation project
+description: The Cloud Native Computing Foundation promoted Flux to Incubation status. We celebrate all the great work that turned Flux into a GitOps family of projects.
+url: /blog/2021/03/flux-is-a-cncf-incubation-project/
+---
+
+![Incubation](/img/incubation.png)
+
+The Flux community is proud to announce that the CNCF Technical
+Oversight Committee just promoted the Flux project to Incubating status!
+🎉
+
+For us as a project, this is proof not only of the wide-spread use of
+Flux (and Helm Operator) v1, but also of our big strides into becoming a
+GitOps family of projects and bringing a toolkit approach to Continuous
+and Progressive Delivery.
+
+We are particularly happy about the support from our community, the
+project maintainers, contributors and end-users who helped us through
+this process. Thanks to our friends at Deutsche Telekom, Lunar,
+MediaMarktSaturn and Sortlist who spoke in favour of Flux and explained
+their GitOps implementation in user interviews with the CNCF.
+
+![Incubation Infographics](/img/incubation-infographics.png)
+
+As a project we have come quite far in the last 5 years. From the time
+when it was a Weaveworks-internal project to make simple deployments
+happen to a buzzing and healthy project with maintainers from various
+companies, hundreds of contributors and a regular release cycle. Thanks
+a lot to [everyone who made Flux
+possible](https://flux.devstats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1)!
+
+> "We created Flux as an open source project because we believe that is
+> the best environment in which to develop software. Since then I have
+> continually been shown that trusting your users and contributors is
+> rewarded. Although it is still a modest project, there is a loyal and
+> growing community around Flux and the methodology it engendered,
+> GitOps. I see the acceptance of Flux into CNCF incubation status as
+> validation of that community as much as the software itself."
+>
+> \-- Michael Bridgen, Flux co-creator
+
+## What's next for Flux
+
+For 10 months we have been working on the new iteration of Flux. It is
+based on modern tooling, very composable and we were able to add
+long-requested features in next to no time.
+
+Flux as a project has been on this trajectory for a longer time already:
+Flux was started in 2016, the Helm Operator was added in 2018, kustomize
+support added in 2019. 2020 was the year in which re-started the project
+to turn Flux into a GitOps family of projects, where simple and focused
+controllers can naturally be composed. 2021 saw Flagger being moved into
+the Flux organization and will see the GA release of Flux v2 in the
+coming months.
+
+## Flux v2 is getting closer
```

Comment:
Somehow "Flux 2 is getting closer" sounds like it's not ready to use? GA helps clarify what we're getting closer to. WDYT?

```suggestion
## Flux v2 getting closer to GA
```